recipes/switcheroo/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "switcheroo" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/switcheroo-{{ version }}.tar.gz
+  sha256: 7963632cd636bb8a70bc3d298d45bd7aff7c11b9bbd55d7336caf9d2fd633557
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python <4.0
+    - pip
+    - poetry
+  run:
+    - python <4.0
```

Need a minimum Python too:

```suggestion
    - python >=3.6,<4.0
    - pip
    - poetry
  run:
    - python >=3.6,<4.0
```